static/src/javascripts/projects/common/modules/atoms/quiz.js

```diff
@@ -21,10 +23,12 @@ define([
 
                 if ($quizzes.length > 0) {
                     bean.on(document, 'click', toArray($quizzes), function (e) {
-                        var quiz = e.currentTarget;
+                        var quiz = e.currentTarget,
+                            total =  $(':checked + .atom-quiz__answer__item', quiz).length;
+
+                        omniture.trackLinkImmediate('quiz-question-answered-'+total);
```

Comment:
@rich-nguyen I think ( looking at clickstream.js, and the fact that i spent about 2 hours failling to get a data-link-name to fire and event, it's because there's no surrounding anchor or button in the html here, so that doesn't work)